### PR TITLE
HTTP-CSS

### PR DESCRIPTION
**Pull request by pswope**
Added answers

### CODE REVIEW

**Comment by pswope** [comment]
Comfort level 4  Plus I typed HTTP vs HTML in the title

File: diagnostic.md [hunk 1]
```diff
@@ -165,6 +175,4 @@ Consider the following media queries. How wide would the screen need to be for e
 }
 ```
 
-<!-- your answer starts here -->
-
-<!-- your answer ends here -->
+2.5 inches
```
Comment (RealWeeks) [inline diagnostic.md:178]: how did you determine inches from px? 600px

File: diagnostic.md [hunk 2]
```diff
@@ -137,11 +149,9 @@ div {
 .divFour {
   clear: left;
 }
-```
-
-<!-- your answer starts here -->
 
-<!-- your answer ends here -->
+divOne will float right
+divFour will float left below the other divThree
```
Comment (RealWeeks) [inline diagnostic.md:155]: what about the other divs?

**Comment by RealWeeks** [comment]
6/7. Good job keep up the good work.

**Pull request closed without merging**